scripts/utils/board_v1_to_v2.py

```diff
@@ -156,19 +156,27 @@ def board_v1_to_v2(board_root, board, new_board, group, vendor, soc, variants):
         board_kconfig_defconfig_file.unlink()
 
     print(f"Creating or updating Kconfig.{new_board}...")
+    board_kconfig_file = new_board_path / "Kconfig.board"
+    with open(board_kconfig_file) as f:
+        for line in f.readlines():
+            if "Copyright" in line:
+                copyright = line
     new_board_kconfig_file = new_board_path / f"Kconfig.{new_board}"
-    selects = "\n\t" + "\n\t".join(["select " + setting for setting in board_soc_settings])
+    header = "# SPDX-License-Identifier: Apache-2.0\n"
+    if copyright is not None:
+        header = header + copyright
```

Comment:
shouldn't this be 
```suggestion
    if copyright is not None:
        header = copyright + header
```
?

Reply:
Right.